### DISCUSSION TITLE
[Scam Alert] usdtpky

### DISCUSSION THREAD

**Discussion by micro73** [Webamoozcom/warning-list#164] [Investigation بررسی]
usdtpky va usdtbang va usdtserins va usdtwen

ادرس سایت :
 usdtpky.com

برای تعدادی از ایرانیان چنین پیامی فرستاده شده 

![1](https://user-images.githubusercontent.com/103103225/184891687-24507902-d640-4667-9ac5-67625a3276ba.jpg)

همه میدانیم چنین پیامک هایی کلاهبرداری است احتمالا با وارد شدن به سایت که اون هم trend micro check هشدار scam page میدهد برای برداشت موجودی گفته شده بگه پول  (کارمزد) واریز کنید تا بتوانید برداشت کنید 
در اینجور مواقع با دریافت کارمزد دیگر پاسخگونیست و شاید درخواست بیشتری پول کنند . 

![22](https://user-images.githubusercontent.com/103103225/184892183-4558249a-6bd5-4e37-8ddb-244aa5156948.jpg)

مراقب و هوشیار باشید.

**Comment by micro73** [comment]
usdtbang.com

![photo_2022-08-18_14-17-19](https://user-images.githubusercontent.com/103103225/185368103-dfff5f89-7861-470e-9196-9cb6ea438f8f.jpg)

این سایت هم شبیه سایت بالایی است و پیامک هایی دقیقا همانند بالایی به مردم ارسال میکنند.

![2022-08-18_14-30-11](https://user-images.githubusercontent.com/103103225/185368575-a70c5e86-a80a-4ab5-aea4-0c75c97a1676.jpg)

**Comment by micro73** [comment]
usdtserins.com

قبلا هم از طریق این دامنه به کلاهبرداری میپرداختند

![WhatsApp Image 2022-08-18 at 4 49 25 PM](https://user-images.githubusercontent.com/103103225/185393420-396cbdb1-f840-4361-b7d6-a1cd497856e7.jpeg)

**Comment by micro73** [comment]
ادرس جدیدشون https://usdtwen.com

**Comment by Kiavash-team** [comment]
سلام دوستان
بنده لیست تمام سایت هایی که دارن چنین کاری و قرار میدم خدمت شما و هشدار وجود داره که در این مورد طمع نکنید و سعی کنید وارد نشید: ⚠

usdtkan.com
usdtcan.com
usdteng.com
usdtbang.com
usdtyun.com
usdtby.com
usdtpky.com
usdtbin.com
usdtbbc.com
usdtong.com
usdtdeng.com
usdtoan.com

این لیست بر اساس سایت هایی که روی آی پی 154.82.110.6 قرار گرفتن و برداشت شدن و در صورت بروزرسانی سایت جدید میتونید به آدرس زیر مراجعه کنید : 

https://dnslytics.com/reverse-ip/154.82.110.6

موفق باشید!

**Comment by Kiavash-team** [comment]
آدرس جدید :❌

usdtzk.com
